docs/fw_binaries.md

```diff
@@ -260,3 +260,33 @@ DM firmware, U-Boot SPL for A53 and device tree blobs
 configuration:
  * path
 
+## For TI AM62Ax devices
+
+[example](../src/snagrecover/templates/am62ax.yaml)
+
+**Warning** Please refer to
+[this documentation](https://software-dl.ti.com/processor-sdk-linux/esd/AM62AX/latest/exports/docs/linux/Foundational_Components/U-Boot/UG-General-Info.html#build-u-boot)
```

Comment:
This link would be useful to have in the docs but since the rest of the doc is identical to the AM62x case, could you just rename the AM62x doc section to AM62x/AM62AX and put this link there?